lib/semian/protected_resource.rb

```diff
@@ -47,8 +48,7 @@ def acquire_bulkhead(timeout, scope, adapter)
         yield self
```

Comment:
Maybe `yield self, 0` here to make the output consistent? Then you can remove `|| 0` above